klippa/src/lib.rs

```diff
@@ -103,3 +118,81 @@ pub trait Subset {
     /// Subset this object. Returns `true` if the object should be retained.
     fn subset(&mut self, plan: &Plan) -> Result<bool, SubsetError>;
 }
+
+pub fn parse_unicodes(unicode_str: &str) -> Result<IntSet<u32>, SubsetError> {
```

Comment:
These two parsing functions should probably be pub(crate) since we don't need to use them outside of this crate. Also suggest moving them into their own module ("cli", "parsing", or "util" maybe?). Could you also add some unit tests for the two parsing functions.